mapping/requirements_officejs.json

```diff
@@ -7812,7 +7812,18 @@
 								}
 							}],
 							"availability": "GA"
-						}						
+						},
```

Comment:
Is there any technical or intentional reason why the DialogApi and IdentityAPI requirement sets are absent for OWA? (since they are present for both Win32 Outlook and W/X/P Online)